README.md

```diff
@@ -1,3 +1,79 @@
 # Mollie PHP Coding Standards
 
 Contains PHP coding standards like rules for PHP-CS-Fixer that serves for purpose of standardization.
+
+## Usage
+
+Place a file named `.php_cs.dist` that has following content in your project's root directory.
```

Comment:
What I miss in all this, is the fact that this doesn't come from this package, but is standard PHP-CS-Fixer functionality. Perhaps divide the first part in 2 subsections:

```
## Usage

This package makes use of PHP-CS-Fixer.

### Already familiar with PHP-CS-Fixer

< short description just pointing out the `Rules::getFor` methods to be used in `->setRules()` >

### New to PHP-CS-Fixer

< what you have now >
```

Reply:
`This package makes use of PHP-CS-Fixer.` This looks fine, but should we really give already familiar, new to information? I think we may assume people already knows, like we did for compser. What do you think?